### PR TITLE
Initialize the system, modules, and test world and run a test against the star wars crawl

### PR DESCRIPTION
**Pull request by bheiskell**
- Copy code base into the foundry modules directory to "install" the module
- Adds helper functions to initialize the FFG system, required modules, and an integration test world
- Adds an integration test for running a opening crawl
- Optionally source cypress baseUrl from cypress.env.json 

Note:
These tests have to be very resilient to race conditions, because the github-ci environment is very overloaded. Avoid waits and force clicks. Force will click the element before it's "ready" per cypress. Sometimes this makes sense, like if one dialog might be covering another, but I think in this case it's a bandaid that's actually triggering more downstream issues. I think the element becomes ready before it's actually clickable.

### CODE REVIEW

**Comment by bheiskell** [comment]
@wrycu This is ready for merge. We'll still need to configure your repo, and we should document that process. We also need to discuss action clean-up and how we want to roll this out, assuming we agree this is worthwhile.

File: cypress/e2e/home.cy.js [hunk 1]
```diff
@@ -1,8 +1,287 @@
...
+      return;
+    }
+    cy.get('#key').type("test-admin-key{enter}");
+  })
+}
+
+/**
+ * Installs module, but it assumes already on the add-on modules tab of setup.
+ */
+function installModule($moduleList, module) {
```
Comment (wrycu) [inline cypress/e2e/home.cy.js:32]: Your other function names describes what it does ("acceptSLicence", "AuthenticateSAsAdmin"), but this does not ("installModule".) Why the change?

**Review by wrycu — APPROVED** [review]
Left a comment but overall LGTM.
